packages/graphile-build-pg/src/plugins/makeProcField.js

```diff
@@ -555,11 +620,16 @@ export default function makeProcField(
                     sql.query`select ${
                       isPgClass
                         ? sql.query`${intermediateIdentifier}.*`
-                        : sql.query`${intermediateIdentifier}.${intermediateIdentifier} as ${functionAlias}`
+                        : isPgRecord
+                          ? sql.query`${intermediateIdentifier}.*`
+                          : sql.query`${intermediateIdentifier} as ${functionAlias}`
```

Comment:
I believe we used the `.*` here for performance reasons - there's an incredible performance difference between `select users from users` and `select users.* from users`. This isn't the kind of thing that typically shows up in tests, but I'd prefer the behaviour of this code didn't change for classes.

Reply:
For classes, the query fragment remains the same.

For scalars, it changed from `${intermediateIdentifier}.${intermediateIdentifier} as ${functionAlias}` to `${intermediateIdentifier} as ${functionAlias}`.

(It would have been clearer to just put `|| isPgRecord` next to `isPgClass`, but I was trying to follow a class-record-scalar pattern for the conditionals.)

Reply:
You're right, I misread the diff.